config/config.json

```diff
@@ -1,8 +1,8 @@
 {
   "development": {
     "username": "root",
-    "password": "vegetable",
-    "database": "exampledb",
+    "password": "Pass@1234",
```

Comment:
You probably don't want to upload your password onto here. I'd remove that and the database name.